Makefile.container_inner

```diff
@@ -0,0 +1,129 @@
+# Copyright 2016--2022 Lightbits Labs Ltd.
```

Comment:
isn't this just a copy of the regular Makefile?

Reply:
> @maor-lb just making sure you are aware, this repo is public and open to the world. I'm not opposed to the lbyaml v2 changes, but we have to make sure the discovery-client remains buildable without it as well for non-LB consumers.

Ack. Do we have CI to test public builds ?

Reply:
Not yet, it's more of a best effort thing right now. Happy to test it when you tell me the PR is ready for testing.